easybuild/easyconfigs/y/yaff/yaff-1.0.develop.2.13-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = "PythonPackage"
+
+name = 'yaff'
+version = '1.0.develop.2.13'
+
+homepage = 'http://molmod.github.io/yaff/'
+description = """Yaff stands for 'Yet another force field'. It is a pythonic force-field code."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['https://github.com/molmod/yaff/releases/download/%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+pyver = '2.7.10'
+
+hdf5ver = '1.8.15-patch1'
+versionsuffix = '-Python-%s-HDF5-%s-serial' %(pyver, hdf5ver)
```

Comment:
please rename this easyconfig to `yaff-1.0.develop.2.13-intel-2015b-Python-2.7.10-HDF5-1.8.15-patch1-serial.eb`, to match the versionsuffix you're using